tests/test_peoplemeasurement.py

```diff
@@ -116,8 +114,8 @@ def create_new_object(timestamp_str="2019-06-21T10:35:46+02:00"):
     )
 
 
-@override_settings(AUTHORIZATION_TOKEN=POST_AUTH_TOKEN)
-@override_settings(GET_AUTHORIZATION_TOKEN=GET_AUTH_TOKEN)
+@override_settings(AUTHORIZATION_TOKEN=settings.AUTHORIZATION_TOKEN)
+@override_settings(GET_AUTHORIZATION_TOKEN=settings.GET_AUTHORIZATION_TOKEN)
```

Comment:
Dit heeft geen zin. Je override nu de setting met zichzelf. 

Zoals je het hiervoor had was top; expliciet een waarde zetten, zodat de context van je testcases voorspelbaar zijn. Als de waarde van de tokens niet van belang is voor een test, dan is de override niet nodig.

Reply:
Klopt. Verwijderd.